### ISSUE TITLE
Health not showing on mobs unless 'always-shown' is set to true

### ISSUE THREAD

**Issue by SimmsRyan97** [DavideBlade/HealthBar-Reloaded#23]
I noticed with mobs that the health isn't appearing unless I have 'always-shown' set to true.

I am using style number 3 (the hearts) but just checked style number 1 for example and it still doesn't work.

Here is my config:
```
mob-bars:
  
  # Choose to enable or disable mob health bar.
  enable: true
  
  # Should the bar be shown on renamed mobs? If so, how?
  # You can choose between "override", "append", and "no".
  show-on-named-mobs: override
  
  # The style for the bar. You can choose between 5 style (1,2,3,4,5).
  display-style: 1
  
  # Always show health bars on all the valid mobs.
  always-shown: false
  
  # Use text instead of bar.
  text-mode: false
  
  # Use your custom text, you will need to set 'text-mode: true'
  custom-text-enable: false
  
  # The custom text. You can use these placeholder:
  # {name} = the name of the mob hit.
  # {health} = the current health.
  # {max} = the maximum health.
  # <3 = a hearth symbol (❤)
  # &4 / &n / &k ... = all the formatting codes.
  #
  # For example this will be displayed as 'Creeper - 16/20'
  custom-text: '{name} - &a{health}/{max}'
  
  # The health bar is shown for the specified time when the mob is hit.
  # In other words, after the time specified by the hit has passed, the bar is hidden again.
  # You will need to set 'always-shown: false'.
  hide-delay-seconds: 5
  
  # If enabled, health bars on mobs will be shown only if the player looks directly at him.
  show-only-if-looking: false
  
  # You can choose to use your own made bars, it will override the previous settings.
  use-custom-file: false
  
  # A list of disabled worlds for mob bars.
  # To enable all worlds, use: disabled-worlds: ''
  disabled-worlds: ''
  
  # These mobs won't show the health.
  # The names are the same in locale.yml.
  # To enable all mobs, use: disabled-types: ''
  disabled-types: ''
```

**Comment by DavideBlade** [comment]
You also get an error in the console right?

**Comment by SimmsRyan97** [comment]
No errors in console.

**Comment by DavideBlade** [comment]
Sorry for the delay. I tried your configuration and did not encounter any problems, neither with `display-style` 1 nor with style 3. I have also tried `always-shown` true and false and everything works correctly. Which version of the plugin and Spigot are you using?
If I had to guess, I'd say there's a plugin that conflicts with HealthBar-Reloaded.

**Comment by SimmsRyan97** [comment]
I am using the latest version of the plugin, the one released on Spigotmc on 4th August. I am using paper 1.21.1.

**Comment by DavideBlade** [comment]
I confirm that everything should work correctly as per the tests I performed. I recommend you try the plugin on a server with only HealthBar to confirm the hypothesis that a plugin you installed is causing some kind of conflict.

**Comment by DavideBlade** [comment]
Since I have not received any updates, I assume the problem has been solved. I am therefore closing the ticket
